targets/csharp/source/source/PlayFabGameserverHttp.cs

```diff
@@ -0,0 +1,72 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace PlayFab
+{
+    // TODO: This should eventually be merged with PlayFabHttp
+    interface IGameserverHttpClient
+    {
+        Task<HeartbeatResponse> SendHeartbeatAsync(HeartbeatRequest request);
+    }
+
+    static class GameserverHttpClientFactory
+    {
+        public static IGameserverHttpClient Instance { get; set; }
+
+        public static IGameserverHttpClient CreateInstance(string baseUrl)
+        {
+            if (Instance == null)
+            {
+                Instance = new GameserverHttpClientProxy(baseUrl);
+            }
+
+            return Instance;
+        }
+    }
+
+    class GameserverHttpClientProxy : HttpClient, IGameserverHttpClient
```

Comment:
Curious: Is Gameserver vs GameServer playfab convention?

Reply:
GameServer

Reply:
Thanks! I'll update that

---
In reply to: [255766688](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r255766688) [](ancestors = 255766688)